integration-tests/src/test/scala-2/chiselTest/QueueFlushSpec.scala

```diff
@@ -230,59 +233,59 @@ class DequeueFullQueueEdgecaseTester(
 
 }
 
-class QueueFlushSpec extends ChiselPropSpec {
+class QueueFlushSpec extends AnyPropSpec with PropertyUtils with ChiselSim {
 
   property("Queue should have things pass through") {
     forAll(vecSizes, safeUIntN(20), Gen.choose(0, 15), Gen.oneOf(true, false)) { (depth, se, tap, isSync) =>
       whenever(se._1 >= 1 && depth >= 1 && se._2.nonEmpty) {
-        assertTesterPasses {
+        simulate {
           new ThingsPassThroughFlushQueueTester(se._2, depth, se._1, tap, isSync)
-        }
+        }(RunUntilFinished(1024 * 10))
```

Comment:
This test randomly dequeues using an LFSR. I set the timeout high enough that it should finish, though I didn't investigate the expected time to finish. The old timeout was 10k.